### PR TITLE
completed basic info page but button still needs to be done

### LINKED ISSUE #7: Basic info in main page🔝

### PR DESCRIPTION
**Pull request by chouche9**

### CODE REVIEW

File: src/assets/img/triangle-left.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<?xml version="1.0" standalone="no"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg class="icon" width="200px" height="200.00px" viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg"><path d="M320 511.872l383.744 383.744V128.128L320 511.872z" fill="#666666" /></svg>
```
Comment (Falanke21) [inline src/assets/img/triangle-left.svg]: Is this file and triangle-right.svg used anywhere in this commit? If no, these .svg files should not be included in this commit.

File: src/pages/Main.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React from "react";
 import { ArrowRightSquare, ArrowLeftSquare } from "react-bootstrap-icons";
 import Heading from "../components/main_page/Heading";
 import GongZhongHao from "../components/main_page/GongZhongHao.js";
+import { Button } from "react-bootstrap";
```
Comment (Falanke21) [inline src/pages/Main.js]: Same with this, it is declared but never used.

File: src/styles/main_page/_Main.scss [hunk 3]
```diff
@@ -1,9 +1,11 @@
 .main-arrow-button {
+    display: flex;
+    align-items: center;
```
Comment (Falanke21) [inline src/styles/main_page/_Main.scss]: Why making this a flex? I don't get why you are changing this arrow button 😂

File: src/styles/main_page/_Common.scss [hunk 4]
```diff
@@ -10,15 +10,33 @@
...
+
+  .intro {
+    font-size: 1.5rem;
+    margin-top: 3.0rem;
+    margin-bottom: 0.8rem;
+    color: $bg_aqua;
   }
 }
+
+.my-button {
```
Comment (Falanke21) [inline src/styles/main_page/_Common.scss]: Same with this. If this code doesn't do anything, we don't need it in the commit

File: src/styles/main_page/_Common.scss [hunk 5]
```diff
@@ -10,15 +10,33 @@
...
+  display: flex;
+  align-items: center;
+}
+
+.right {
```
Comment (Falanke21) [inline src/styles/main_page/_Common.scss]: Same with this.

**Review by Falanke21 — CHANGES_REQUESTED** [review]

**Comment by Falanke21** [comment]
Besides the code style, the layout looks very pretty to me. 👍

**Comment by Falanke21** [comment]
👮‍♂️💪👍

**Review by rainyuxuan — COMMENTED** [review]
I changed some code structure in the files, please continue working on Frank's review.

**Comment by rainyuxuan** [comment]
The problem I made during the last meeting was caused by the unwatched sass. It is now solved and I removed the css and css.map files. 😅

**Review by rainyuxuan — APPROVED** [review]
All solved

**Review by Falanke21 — APPROVED** [review]
I will merge this, but the mobile-adaption still needs to be done

**Pull request merged**